clusters/app.ci/assets/dptp-controller-manager.yaml

```diff
@@ -152,6 +152,7 @@ spec:
         - --testImagesDistributorOptions.additional-image-stream-tag=ocp/builder:rhel-golang-1.11
         - --testImagesDistributorOptions.additional-image-stream-tag=ocp/builder:rhel-golang-1.12
         - --testImagesDistributorOptions.additional-image-stream-tag=ocp/builder:rhel-golang-1.13
+        - --testImagesDistributorOptions.additional-image-stream-tag=ci/ci-operator:latest
```

Comment:
If this controller creates the imagestream, it will have `local: false` and hence not work for pods that have `ci-operator:latest` als image. Should the controller set the `local: true`? It can not be set on the import it seems

Reply:
Why will the controller make it that way? Can't we just set the thing in the api call to create?

Reply:
I don't understand the comment?